### PR TITLE
fix#1460 - Message screen shown when no questions are returned

### LINKED ISSUE #1460: Help Fragment: blank screen is shown when no questions are found
**Summary:**

In Help Fragment, a blank screen is shown if no questions are found after the search query. Like other parts in the application, we can make use of `SweetError` library to handle this scenario by displaying a message: "No questions found" 

**Steps to reproduce:**

* Go to Help fragment from Navigation drawer
* Write a search query so that no resultant questions are returned
* Just a blank a screen should be displayed

**Expected behaviour:**

A message like "No Questions Found" should have been displayed

**Observed behaviour:**

A blank screen is being displayed

**Screenshots:**

WhatsApp Image 2020-04-24 at 11 24 58 PM

### PR DESCRIPTION
**Pull request by devansh-299**
Fixes #1460 

**Screenshot:**

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Review by ShivangiSingh17 — APPROVED** [review]

**Comment by miPlodder** [comment]
@devansh-299 Nice job. Can you fix the conflict?

**Comment by devansh-299** [comment]
> 
> 
> @devansh-299 Nice job. Can you fix the conflict?

Sure! I will update the PR with the required changes

**Comment by devansh-299** [comment]
> 
> 
> @devansh-299 Nice job. Can you fix the conflict?

Done

File: app/src/main/java/org/mifos/mobile/ui/fragments/HelpFragment.kt [hunk 1]
```diff
@@ -121,12 +129,25 @@ import javax.inject.Inject
             }
 
             override fun onQueryTextChange(newText: String): Boolean {
-                faqAdapter?.updateList(presenter?.filterList(faqArrayList, newText))
+                val filteredList = presenter?.filterList(faqArrayList, newText)
```
Comment (miPlodder) [inline app/src/main/java/org/mifos/mobile/ui/fragments/HelpFragment.kt]: Let's use a better choice of variable names like `filteredFAQList` or something better.

File: app/src/main/java/org/mifos/mobile/ui/fragments/HelpFragment.kt [hunk 2]
```diff
@@ -121,12 +129,25 @@ import javax.inject.Inject
...
+                    } else {
+                        showEmptyUI()
+                    }
+                } ?: showEmptyUI()
                 return false
             }
         })
     }
 
+    private fun showEmptyUI() {
```
Comment (miPlodder) [inline app/src/main/java/org/mifos/mobile/ui/fragments/HelpFragment.kt]: Let's use a better name for this `method/function`, it appears to be too generic. for instance `showEmptyFAQUI` or something better

**Review by miPlodder — CHANGES_REQUESTED** [review]
If possible, re-size image/screenshots before adding them in a comment for PR. (use any photo editing tools, or use this https://pixlr.com/e/). Do follow this in the future also. 
If your issues solve a flow, do add multiple screenshots or GIF. As I see that's not done here. Do follow these standard conventions

Take care of the comments given. Start using better choices of variables/method names in the future. (behavior/functionality and naming should go hand-in-hand). Rest seems cool to me

File: app/src/main/java/org/mifos/mobile/ui/fragments/HelpFragment.kt (hunk 1 above)
Comment (devansh-299) [reply app/src/main/java/org/mifos/mobile/ui/fragments/HelpFragment.kt]: Oh thanks, I will the PR with the required changes

**Comment by devansh-299** [comment]
> 
> 
> If possible, re-size image/screenshots before adding them in a comment for PR. (use any photo editing tools, or use this https://pixlr.com/e/). Do follow this in the future also.
> If your issues solve a flow, do add multiple screenshots or GIF. As I see that's not done here. Do follow these standard conventions
> 
> Take care of the comments given. Start using better choices of variables/method names in the future. (behavior/functionality and naming should go hand-in-hand). Rest seems cool to me

Thank you for the suggestions, I have updated the pull request description, and will update the pull request asap

**Pull request merged**